legacy/bootloader/combine_upgrade_firmware.py

```
@@ -0,0 +1,336 @@
+#!/usr/bin/env python3
```

Ruff (0.14.6)

Shebang is present but file is not executable

(EXE001)